site_config_client/__init__.py

```diff
@@ -90,13 +93,10 @@ def get_config(self, site_uuid: Union[str, uuid.UUID],
         """
         Returns a single configuration object for Site
         """
-        endpoint = 'v1/configuration/{}/'.format(site_uuid)
-        data = {
-            "type": type,
-            "name": name,
-            "status": status
-        }
-        response = requests.get(self.build_url(endpoint), data=data)
+        endpoint = 'v1/configuration/{}/?type={}&name={}&status={}'.format(site_uuid, type, name, status)
```

Comment:
**suggestion:** instead of using `requests.get('/url?a=1&b=2')` we should use the following format `requests.get('/url', data={'a': 1, 'b': 2})` becasue the latter takes care of encoding the params which saves us an entire class of errors and security issues:

 - https://medium.com/disruptive-labs/a-quick-primer-on-encoding-decoding-for-security-folks-a021afd98fbe
 - https://www.programcreek.com/python/example/3765/requests.get